Gifski.xcodeproj/xcshareddata/xcschemes/Gifski.xcscheme

```diff
@@ -54,6 +54,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      enableAddressSanitizer = "YES"
```

Comment:
do we want to keep the sanitizer enabled?

Reply:
Yes, it's the one that helped me track down the issue. It adds a slight overhead, but only in debug. I think it's worth it.